### ISSUE TITLE
Request for Final Exam

### ISSUE THREAD

**Issue by jhyearsley** [uwhpsc-2016/syllabus#41]
@cswiercz Since the final exam will be mostly the same format as the quizzes, will we be required to answer each question before moving on to the next one as has been the case for the quizzes? I hope that is not the case since it is a larger exam and in general it's a good test taking strategy to jump around from question to question!

**Comment by cswiercz** [comment]
No. All quiz questions will be presented at the same time. Thank you for asking.
